juniper_official/interface/interface-error-details.yml

```diff
@@ -28,6 +29,9 @@ intfStatsView:
     fec_uncorr_err: {fec_nccw_count: int}
     fec_corr_err_rate: {fec_ccw_error_rate: int}
```

Comment:
This file has too many fields which are not used in the rules. Can you please remove them which are not required by the rules?

Reply:
Done